sample/reverse-identifiers-async.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Example jscodeshift async transformer. Simply reverses the names of all
+ * identifiers.
+ */
+ function transformer(file, api) {
```

Comment:
Make this `async` rather than returning a resolved promise? I think that'll more accurately reflect how people use it.